imagebuildah/build.go

```diff
@@ -1116,6 +1117,16 @@ func (b *Executor) Commit(ctx context.Context, ib *imagebuilder.Builder, created
 	b.builder.SetEntrypoint(config.Entrypoint)
 	b.builder.SetShell(config.Shell)
 	b.builder.SetStopSignal(config.StopSignal)
+	b.builder.SetHealthcheck(nil)
+	if config.Healthcheck != nil {
+		b.builder.SetHealthcheck(&buildahdocker.HealthConfig{
+			Test:        append([]string{}, config.Healthcheck.Test...),
```

Comment:
Why do you need this as append?

Can't you just do 
Test: config.Healthcheck.Test

Reply:
This copies the slice, which I prefer to do instead of referencing the passed-in one, which can be changed (intentionally or not) after we return.